unity/Assets/Scripts/PhysicsRemoteFPSAgentController.cs

```diff
@@ -3960,10 +3960,39 @@ public struct WhatDidITouch {
             }
 
             SimObjPhysics[] simObjs = GameObject.FindObjectsOfType(typeof(SimObjPhysics)) as SimObjPhysics[];
-            foreach (SimObjPhysics sop in simObjs) {
-                if (sop.Type.ToString() == objectType) {
+            if(simObjs != null) {
+                foreach (SimObjPhysics sop in simObjs) {
+                    if (sop.Type.ToString() == objectType) {
+                        if (sop.DoesThisObjectHaveThisSecondaryProperty(SimObjSecondaryProperty.CanBreak)) {
+                            //look both in this object and children so things like Windows don't FREAK OUT
+                            sop.GetComponentInChildren<Break>().Unbreakable = true;
+                        }
+                    }
+                }
+            }
+
+            actionFinished(true);
+        }
+
+        public void MakeAllObjectsUnbreakable() {
+            SimObjPhysics[] simObjs = GameObject.FindObjectsOfType(typeof(SimObjPhysics)) as SimObjPhysics[];
+            if(simObjs != null) {
+                foreach (SimObjPhysics sop in simObjs) {
+                    if (sop.DoesThisObjectHaveThisSecondaryProperty(SimObjSecondaryProperty.CanBreak)) {
+                        sop.GetComponentInChildren<Break>().Unbreakable = true;
+                    }
+                }
+            }
+
+            actionFinished(true);
+        }
+
+        public void MakeAllObjectsBreakable() {
```

Comment:
`MakeAllObjectsUnbreakable` and `MakeAllObjectsUnbreakable` have basically the same logic. Can we instead have a `UpdateBreakabilityOfAllObjects` function that takes a `breakable` boolean as input?